lib/src/models/twitch_clip.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'twitch_clip.freezed.dart';
+part 'twitch_clip.g.dart';
+
+@freezed
+class TwitchClip with _$TwitchClip {
+  const factory TwitchClip({
+    @JsonKey(name: 'id') required String id,
+    @JsonKey(name: 'url') required String url,
```

Comment:
`@JsonKey` is not needed:
```suggestion
    required String id,
    required String url,
```